cli/js/globals.ts

```diff
@@ -234,6 +235,13 @@ export const windowOrWorkerGlobalScopeProperties = {
   Worker: nonEnumerable(workers.WorkerImpl),
 };
 
+export const windowGlobalScopeProperties = {
```

Comment:
While technically, web workers don't have access to local storage, I would not want to add another bag here.  We already have things in worker global that shouldn't technically be there.  @bartlomieju opinions?

Reply:
I agree with you @kitsonk - workers shouldn't have access to `localStorage`